src/Compatibility/Core/src/Android/NativeBindingservice.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Maui.Controls.Compatibility.Platform.Android
 	class NativeBindingService : INativeBindingService
 	{
 		[UnconditionalSuppressMessage("Trimming", "IL2075", Justification = TrimmerConstants.NativeBindingService)]
+		[UnconditionalSuppressMessage("Trimming", "IL2026", Justification = TrimmerConstants.NativeBindingService)]
```

Comment:
I'm not very happy we have these `UnconditionalSuppressMessage` here. I think in this case it might be worth putting `[RequiresUnreferencedCode]` on the `INativeBindingService.TrySetBinding(object, string, BindingBase)` and all the classes that implement it.